### PR TITLE
feat(AWS Lambda): Support `ephemeralStorageSize`

### PR DESCRIPTION
**Pull request by pgrzesik**
Closes: #10899

### CODE REVIEW

File: docs/providers/aws/guide/functions.md [hunk 1]
```diff
@@ -680,6 +680,17 @@ functions:
         - subnetId1
 ```
 
+## Size of available ephemeral storage
+
+You can configure size of the ephemeral storage available to your Lambda function under `/tmp` directory by setting the `ephemeralStorageSize` property in the function configuration in `serverless.yml`. It should be a numerical value of MBs, between 512 and 10240. If you don't specify any configuration, the function will have 512 MB ephemeral storage available by default.
```
Comment (mnapoli) [inline docs/providers/aws/guide/functions.md:685]:
```suggestion
By default, Lambda [allocates 512 MB of ephemeral storage](https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-ephemeral-storage) in functions under the `/tmp` directory.

You can increase its size via the `ephemeralStorageSize` property. It should be a numerical value in MBs, between 512 and 10240.
```

File: docs/providers/aws/guide/functions.md [hunk 2]
```diff
@@ -680,6 +680,17 @@ functions:
         - subnetId1
 ```
 
+## Size of available ephemeral storage
```
Comment (mnapoli) [inline docs/providers/aws/guide/functions.md:683]:
```suggestion
## Ephemeral storage
```

(for simplicity's sake?)

File: docs/providers/aws/guide/functions.md (hunk 1 above)
Comment (mnapoli) [reply docs/providers/aws/guide/functions.md:685]: (I reused some of the wording in https://docs.aws.amazon.com/lambda/latest/dg/configuration-function-common.html#configuration-ephemeral-storage)

File: docs/providers/aws/guide/functions.md (hunk 2 above)
Comment (pgrzesik) [reply docs/providers/aws/guide/functions.md]: 👍

File: docs/providers/aws/guide/functions.md (hunk 1 above)
Comment (pgrzesik) [reply docs/providers/aws/guide/functions.md]: 👍

**Review by mnapoli — APPROVED** [review]

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**